code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -134,8 +134,9 @@
 							blood_splatter(target, null, 1)
 			blood_splatter(src.loc, null, 1)
 			playsound(src.loc, 'sound/effects/splat.ogg', 50, 1, -6)
-			H.bloody_hands()
-			if (src.loc == usr)
+			if(ishuman(H))

```

Comment:
This can just be `istype(H)`, and you also duplicate this check below.
Perhaps:
```
if (istype(H))
    H.bloody_hands()
    if (loc == usr)
        H.bloody_body()
```